website/docs/reference/snapshot-configs.md

```diff
@@ -26,9 +20,7 @@ Parts of a snapshot:
 
 <VersionBlock lastVersion="1.8">
 
-import SnapshotYaml from '/snippets/_snapshot-yaml-spec.md';
-
-<SnapshotYaml/>
+Starting from Versionless and Core v1.9, defining snapshots in a `.sql` file using a config block is a legacy method. You can define snapshots in YAML format using the latest [snapshot-specific configurations](/docs/build/snapshot-configs#snapshot-specific-configurations). For new snapshots, we recommend using these latest configs. If applying them to existing snapshots, you'll need to migrate over.
```

Comment:
Including this section for v1.8 creates awkwardness because then the "The following resource-specific configurations are only available to Snapshots:" sentence is just hanging there:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/c5e4de00-52a5-45c3-a958-143815e7fe1c">

---

Do you have any ideas to make it look better for v1.8?

One idea: put it _within_ the "Config block" tab.